### PR TITLE
Server game setup hopefully

### PR DESCRIPTION
**Pull request by andersonn**

### CODE REVIEW

**Comment by ftommasi** [comment]
Thumbs up!

**Comment by ftrelz** [comment]
Did you actually call start on server game?

**Comment by andersonn** [comment]
Will do that the minute I get home.

**Comment by ftrelz** [comment]
Ok, recommit those changes and then I think we can merge this.

File: Server/GameObjectManager.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+};  
+  
+GameObjectManager.prototype.AddObject = function (object) {
+    this.GameObjectList.push(object);
+};
+
+GameObjectManager.prototype.UpdateAll = function (delta) {
+  //console.log('From GO Manager ' + this.GameObjectList.length);
+  for (var i = 0; i < this.GameObjectList.length; i++)
+    //TODO (EVERYONE) We need to type everything so I can go by class in the gameObjectList
```
Comment (andersonn) [inline Server/GameObjectManager.js:20]: Note this todo statement.

**Pull request merged**